SU-SWS/sulgryphon-nextjs
# Keyword search filter stubbed out

# READY FOR REVIEW
- Updated the styling to match the designs.

#TO TEST 
- Add a person list on a basic page.
- Verify the search box matches the design in Figma.
- Review code.
[- SUL23-503](https://stanfordits.atlassian.net/browse/SUL23-503)


# Summary
- Stubbed out keyword filtering functionality.
- Needs styling - done.
